include/arbor/time_sequence.hpp

```diff
@@ -33,9 +32,10 @@ class time_seq {
 
     template <
         typename Impl,
-        typename = typename util::enable_if_t<
+        typename = typename std::enable_if<
```

Comment:
gargh. Bring on C++14.

Reply:
Oh yep. Next on the code quality TODO list.